components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/listener/IDPMgtAuditLogger.java

```diff
@@ -69,14 +69,20 @@ public boolean doPostUpdateIdP(String oldIdPName, IdentityProvider identityProvi
 
     @Override
     public boolean doPostDeleteIdP(String idPName, String tenantDomain) throws IdentityProviderManagementException {
-        if(StringUtils.isEmpty(idPName)){
+        if (StringUtils.isEmpty(idPName)) {
             idPName = "Undefined";
         }
         audit.info(String.format(AUDIT_MESSAGE, getUser(), "delete", UserCoreUtil.addTenantDomainToEntry
                 (idPName, tenantDomain), null, SUCCESS));
         return true;
     }
 
+    @Override
+    public boolean doPostDeleteIdPs(String tenantDomain) throws IdentityProviderManagementException {
```

Comment:
Shall we add the method comment here? If so please check the other places as well.